<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AutoTool</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>自动化测试工具</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,3,4">
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>测试模式</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,1,1">
           <item>
            <widget class="QRadioButton" name="mucButton">
             <property name="text">
              <string>MCU压力测试</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QRadioButton" name="appButton">
             <property name="text">
              <string>App自动化测试</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>串口(COM?)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="serialEdit">
             <property name="placeholderText">
              <string>必填项！！！</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>测试次数</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="countEdit">
             <property name="placeholderText">
              <string>必填项！！！</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>次</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,1,1">
             <item>
              <widget class="QCheckBox" name="mcuBox">
               <property name="text">
                <string>MCU压测(模式,夜灯..)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="add_deviceBox">
               <property name="text">
                <string>添加/删除设备</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="add_tempBox">
               <property name="text">
                <string>绑定/解绑温湿度计</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>测试数据</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;！！！切记串口已输入密码(govee),并且已经关闭多余打印(log print 0)！！！&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="textFormat">
                <enum>Qt::AutoText</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="mcuTextEdit">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="placeholderText">
                <string>输入mcu指令(iot_test 01 01,iot_test 01 00)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="assertTextEdit">
               <property name="placeholderText">
                <string>输入断言数据(开机:55 11 01 00 01 01 69, 关机:55 11 01 00 01 00 68)</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>开始测试</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="stopButton">
             <property name="text">
              <string>暂停测试</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>测试结果</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QTextBrowser" name="resultBrowser"/>
           </item>
           <item>
            <widget class="QPushButton" name="clearButton">
             <property name="text">
              <string>清除</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="quitButton">
             <property name="text">
              <string>退出</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
